js/weather-api/index.html

```diff
@@ -7,6 +7,17 @@
 </head>
 
 <body>
+    <select id="selectCountry">
+        <option value="london" selected>ロンドン</option>
+        <option value="tokyo">東京</option>
+        <option value="pali">パリ</option>
+        <option value="new%20york">ニューヨーク</option>
+        <option value="indonesia">インドネシア</option>
```

Comment:
都市はジャカルタを指定しましょう。
これはインドネシアと書いている問題文が悪い気がしますね。出題者と相談します。
(編集)
出題者に、問題文のインドネシアをジャカルタに修正してもらいました。お手数ですが、ジャカルタでお願いします。

```suggestion
        <option value="jakarta">ジャカルタ</option>
```